crates/ra_ide/src/call_info.rs

```diff
@@ -51,36 +51,39 @@ pub(crate) fn call_info(db: &RootDatabase, position: FilePosition) -> Option<Cal
     // If we have a calling expression let's find which argument we are on
     let num_params = call_info.parameters().len();
 
-    if num_params == 1 {
-        if !has_self {
-            call_info.active_parameter = Some(0);
-        }
-    } else if num_params > 1 {
-        // Count how many parameters into the call we are.
-        if let Some(arg_list) = calling_node.arg_list() {
-            // Number of arguments specified at the call site
-            let num_args_at_callsite = arg_list.args().count();
-
-            let arg_list_range = arg_list.syntax().text_range();
-            if !arg_list_range.contains_inclusive(position.offset) {
-                tested_by!(call_info_bad_offset);
-                return None;
+    match num_params.cmp(&1) {
```

Comment:
Seems like this becomes less redable, mainly because we are comparing with a constant `1`, and `Ordering` obsurres the fact that rhs is a small constal. I guess the best solution here is just a match?

```rust
match num_params {
  0 => (),
  1 => { ... }
  _ => { ... }
}
```